src/index.js

```diff
@@ -142,13 +142,22 @@ yargs
       yargs.positional('txHash', { required: true })
     },
     argv => {
-      const getTransactionObject = require('./getTransactionObject')
-
+      const { getTransaction, getReceipt } = require('./getTransactionObject')
       const { txHash, url } = argv
+      const promises = [getTransaction(txHash, url), getReceipt(txHash, url)]
 
-      getTransactionObject(txHash, url).then(transactionObj => {
-        console.log(JSON.stringify(transactionObj, null, 2))
-      })
+      promises
+        .reduce((promiseChain, currentTask) => {
```

Comment:
Is the reduce necessary? Can't we just use a `Promise.all`?

```javascript
Promise.all(promises).then(([transaction, receipt]) => {
  transaction.receipt = receipt
  ...
})

Reply:
Done!